src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowLipPredicate.java

```diff
@@ -11,7 +11,7 @@ private FlowLipPredicate() {
     }
 
     public static final Predicate<CaseData> isLipCase = caseData ->
-        caseData.isApplicantNotRepresented() || caseData.isApplicantRepresented();
+        caseData.isApplicantNotRepresented();
```

Comment:
Should we also check that the defendant is LiP? Maybe it would be better to use the isLipvLipOneVOnede caseData method.